test/src/server/routes/type-editor/relationship-type.js

```diff
@@ -0,0 +1,188 @@
+import {createEditor, createRelationshipType, truncateEntities} from '../../../../test-helpers/create-entities';
+import app from '../../../../../src/server/app';
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import orm from '../../../../bookbrainz-data';
+
+
+const {RelationshipType} = orm;
+
+chai.use(chaiHttp);
+const {expect} = chai;
+
+describe('Relationship Type routes with Relationship Editor priv', () => {
+	const id1 = 1;
+	let agent;
+	before(async () => {
+		try {
+			await createRelationshipType(id1);
+			await createEditor(123456, 4);
+		}
+		catch (error) {
+			// console.log(error);
+		}
+		// Log in; use agent to use logged in session
+		agent = await chai.request.agent(app);
+		await agent.get('/cb');
+	});
+	after(truncateEntities);
+
+	it('should not throw an error if creating new relationship type', async () => {
+		const res = await agent.get('/relationship-type/create');
+		expect(res.ok).to.be.true;
+		expect(res).to.have.status(200);
+	});
+
+	it('should not throw an error trying to edit an existing relationship type', async () => {
```

Comment:
This could be confusing. At first I was surprised this worked, because I was expecting editing a rel type without any changes to throw an error. Then I realized we're not actually editing anything here, just accessing the page.
Can you go through the test names in this file again, and apply the same change wherever it could be confusing?
```suggestion
	it('should not throw accessing the relationship type editor', async () => {
```

Reply:
fixed!